Sources/Public/Animation/LottieAnimationView.swift

```diff
@@ -676,6 +676,10 @@ open class LottieAnimationView: LottieAnimationViewBase {
   /// Reads the value of a property specified by the Keypath.
   /// Returns nil if no property is found.
   ///
+  /// Note: This method isn't supported by the Core Animation rendering engine and will always return `nil` if used.
+  /// It is still supported by the Main Thread rendering engine. You can read more about it at ``currentRenderingEngine``,
+  /// and it can be set through the view's ``configuration``.
+  ///
```

Comment:
I think we can leave this as:

```suggestion
  /// Note: This method isn't supported by the Core Animation rendering engine and will always return `nil` if used.
  /// It is still supported by the Main Thread rendering engine.
  ///
```